### PR TITLE
Revert profile injection for VcLdpManager on vc-api endpoints

### PR DESCRIPTION
**Pull request by PatStLouis**
Pretty straightforward, I changed the way these endpoint instantiate the `VcLdpManager`. This is to address [this issue](https://github.com/hyperledger/aries-cloudagent-python/issues/2793)

### CODE REVIEW

**Comment by PatStLouis** [comment]
@dbluhm could I get a quick approval?

**Comment by swcurran** [comment]
@jamshale — over to you for approval. Let’s leave @dbluhm and the baby be for a while 👶 🍼

**Comment by PatStLouis** [comment]
oh wow, here's some news I wasn't aware of! Congratulations @dbluhm !

**Review by jamshale — APPROVED** [review]

**Comment by jamshale** [comment]
Thank you. Approved. Hopefully it wasn't leveraged anywhere else. There was only a few PR's after it was merged so I'm pretty sure this must be it.

**Pull request merged**
